Fix allow users to delete their comments posted on SNS.

tkhsknsk opened the pull request:
When you click the x button, the action disappears.
However, reloading the feeds screen will restore them.
There is some cache left.

https://github.com/s-tip/stip-sns/issues/33

tkhsknsk committed:
Fix allow users to delete their comments posted on SNS.

s-tip/stip-sns#33

stmtstk commented:
Thank you!! @tkhsknsk 

I checked the deletion behaviors, it seems to work well.
However, some cache files are not deleted.

<Case 1: deletion of the post with attachment files>
The STIX files made from attachment files are left in a cache and an attach directory.

<Case 2: delete of the post with comments>
The STIX files made from comment are left in a cache  directory.

We should delete these files to avoid a disk full issue.

----

投稿削除時の表示関連、 mongo, mysql  などの database 周りは削除されているので動作としては正しいと思います。
投稿以外の STIX FIles、この場合はコメントと添付ファイルが cache と attache dir から削除されていませんでした。
動作には問題ないのですが、削除しないことでいずれおきるであろう disk full を避けるためにこの issues で削除対応をしたいです。

tkhsknsk committed:
Source Review Accepted.

- Fix to raise exception when deleting files
- Variable rename, l -> list_ or str_
- Deleting Unused Variables

tkhsknsk committed:
Fix cache file remove

Correcting an undeleted STIX cache file in an attached file

stmtstk commented:
@tkhsknsk Thank you!
However, I would like to remove print debugs in src/feeds/views.py.

tkhsknsk committed:
remove print debug

stmtstk commented on src/feeds/views.py:
```diff
@@ -723,12 +723,9 @@ def remove(request):
                 remove_file_name_id = rs.convert_package_id_to_filename(remove_package_id)
                 remove_path = Feed.get_cached_file_path(remove_file_name_id)
-                print(remove_path)
                 os.remove(remove_path)
```
We should be only `"os.remove(remove_path)"` between try and except.

```
for remove_package_id in remove_package_ids:
    remove_flie ..
    remove_path ...
    try:
        os.remove(remove_path)
    except FileNotFoundError:
        pass

```

stmtstk replied:
Good. Thanks.

tkhsknsk committed:
Change Try Indent

stmtstk commented:
Thanks @tkhsknsk.
We can close this issue.
